Makefile

```diff
@@ -39,3 +39,19 @@ wire:
 run-worker:
 	@go run cmd/worker/wire_gen.go cmd/worker/worker.go
 
+.PHONY: run-management-api
+run-management-api:
+	@go run cmd/management_api/wire_gen.go cmd/management_api/management_api.go
+
+.PHONY: generate
+generate: buf/install
+	@go generate ./gen
+
+.PHONY: buf/install
+buf/install:
```

Comment:
Can't we move those dependencies definitions to our `tools/` package? In this way, we'll also fix the version we're using.